File: test/layouttest.cpp
#include "paragui.h"
#include "pgapplication.h"
#include "pgwindow.h"

int main() {

	// construct the application object
	PG_Application app;

	app.EnableBackground(true);
	app.LoadTheme("default");
	
	// init 1024x768
	if(!app.InitScreen(800,600, 0, SDL_SWSURFACE)){
		printf("Resolution not supported\n");
		exit(-1);
	}

	PG_Window wnd(NULL, PG_Rect(0,0,400,300), "Test");
	wnd.LoadLayout("pokus.xml");

	SDL_Delay(2000);
	wnd.Show();
	
	// Enter main loop 
	app.Run();

	return EXIT_SUCCESS;
}

File: bindings/ruby/layouttest.rb
# test XML layout

require 'paragui'

include Paragui

app = PG_Application.new
app.EnableBackground true
app.LoadTheme "default"

if not app.InitScreen(800, 600, 0, 0)
  puts "Reesolution not supported!"
  exit
end

wnd = PG_Window.new nil, PG_Rect.new(0,0,400,300), "Test"
wnd.LoadLayout "pokus.xml"

sleep 2
wnd.Show

app.Run
